tests/test_loads.py

```diff
@@ -50,76 +50,71 @@ def run_gdb_with_script(binary='', core='', pybefore=None, pyafter=None):
     return output
 
 
-HELLO = (
-    'pwndbg: loaded ### commands. Type pwndbg [filter] for a list.\n'
-    'pwndbg: created $rebase, $ida gdb functions (can be used with print/break)\n'
-)
+HELLO = [
+    'pwndbg: loaded ### commands. Type pwndbg [filter] for a list.',
+    'pwndbg: created $rebase, $ida gdb functions (can be used with print/break)'
+]
 
 BASH_BIN = tests.binaries.old_bash.get('binary')
 BASH_CORE = tests.binaries.old_bash.get('core')
 
-launched_locally = not(os.environ.get('PWNDBG_TRAVIS_TEST_RUN'))
+launched_locally = not (os.environ.get('PWNDBG_TRAVIS_TEST_RUN'))
 
 
 def test_loads_pure_gdb_without_crashing():
-    output = run_gdb_with_script()
+    output = run_gdb_with_script().splitlines()
     assert output == HELLO
 
 
 @pytest.mark.skipif(launched_locally, reason='This test uses binaries compiled on travis builds.')
 def test_loads_binary_without_crashing():
-    output = run_gdb_with_script(binary=BASH_BIN)
+    output = run_gdb_with_script(binary=BASH_BIN).splitlines()
 
-    expected = 'Reading symbols from %s...(no debugging symbols found)...done.\n' % BASH_BIN
+    expected = ['Reading symbols from %s...(no debugging symbols found)...done.' % BASH_BIN]
     expected += HELLO
 
-    assert output == expected
+    assert all(item in output for item in expected)
 
 
 @pytest.mark.skipif(launched_locally, reason='This test uses binaries compiled on travis builds.')
 def test_loads_binary_with_core_without_crashing():
-    output = run_gdb_with_script(binary=BASH_BIN, core=BASH_CORE)
-
-    expected = 'Reading symbols from %s...(no debugging symbols found)...done.\n' % BASH_BIN
-    expected += '''[New LWP 13562]
-Core was generated by `/home/user/pwndbg/tests/corefiles/bash/binary'.
-Program terminated with signal SIGINT, Interrupt.
-#0  0x00007ffff76d36b0 in faccessat (fd=0, file=0x7fffffffc8ef "'''
-    assert output.startswith(expected)
-
-    # Skip 4 characters as this is some random thing
-    output = output[len(expected)+10:]
-    expected = '''", 
-    mode=1, flag=-1) at ../sysdeps/unix/sysv/linux/faccessat.c:41
-41	../sysdeps/unix/sysv/linux/faccessat.c: No such file or directory.
-'''
-
+    output = run_gdb_with_script(binary=BASH_BIN, core=BASH_CORE).splitlines()
+
+    expected = [
+        'Reading symbols from %s...(no debugging symbols found)...done.' % BASH_BIN,
+        '''[New LWP 13562]''',
+        '''Core was generated by `/home/user/pwndbg/tests/corefiles/bash/binary'.''',
+        '''Program terminated with signal SIGINT, Interrupt.''',
+        '''#0  __addmntent (stream=0x7ffff79a7640 <_nl_global_locale+192>, ''',
+        '''    mnt=<optimized out>) at mntent_r.c:257''',
+        '''257\tmntent_r.c: No such file or directory.'''
+    ]
     expected += HELLO
 
-    assert output == expected
+    assert all(item in output for item in expected)
```

Comment:
All that asserts has the `all()` function which search every expected element in the output.